lib/eslint.js

```diff
@@ -600,6 +617,9 @@ module.exports = (function() {
         // process initial config to make it safe to extend
         config = prepareConfig(config || {});
 
+        // check config for missing rules
+        verifyRulesExist(config);
```

Comment:
We are iterating over all rules 2 times now.
Can we do these checks when we are iterating over the rules at line 647? just to avoid double iteration.